CMakeLists.txt

```diff
@@ -10,31 +10,34 @@ if(NOT SONATA_VERSION)
                     OUTPUT_STRIP_TRAILING_WHITESPACE)
     if(GIT_VERSION_FAILED)
       message(
-        FATAL_ERROR
+        STATUS
```

Comment:
I will change SONATA_VERSION to SONATA_REPORT_VERSION ? because when this will become submodule then there will be conflict?